README.md

```diff
@@ -145,32 +145,12 @@ Thanks to everyone whose idea and codebase was used in this project🙏
 
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
 
-
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 📝 License <a name="license">MIT License</a>
 
-## 📝 License <a name="license">MIT License
-
-This project is [MIT](Copyright (c) [2023] [Kevin Okoth]
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL] The link to the MIT license in your README file is broken. I think this happened while trying to implement the optional suggestion from the previous reviewer. I would have made this to be required this time around if i